compiler/src/iree/compiler/Codegen/Common/GPUPipelining.cpp

```diff
@@ -23,6 +25,49 @@ namespace iree_compiler {
 static const StringLiteral kPipeliningLoopMarker = "__pipelining_K_loop__";
 static const StringLiteral kPipeliningGlobalLoad = "__pipelining_global_load__";
 
+
+// Returns a new AsyncCopyOp with Zfill 
+static Operation* replaceAsyncCopywithAsyncCopyZfill(Operation* op, Value pred, PatternRewriter& rewriter) {
```

Comment:
nit: the naming is a bit confusing, I think it should be something explicitly saying that it predicates operations

Reply:
Done. Renamed to `replaceOpWithPredicatedOp`